.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "semi": false,
+  "trailingComma": "all"
```

Comment:
this is the only nit I have, I'm personally not a fan of the trailing comma though I will defer to the team's majority vote. @floating what's your stance?

Reply:
Removing trailing commas, it's often a contentious one.

Reply:
Not really a fan of single quote in JSX tbh, however I can see both sides of the argument - "mirror HTML preferences" vs "JSX is not HTML".  I think the double quotes help to visually distinguish between JSX string literals, string literals elsewhere and template literals in JSX.  Or maybe it's just what I'm used to - React docs feature double-quoted JSX and it's the default in ESLint.  Everywhere else, there's debate:

https://github.com/airbnb/javascript/issues/629
https://github.com/airbnb/javascript/issues/269#issuecomment-134990455
https://github.com/xojs/eslint-config-xo/pull/78
https://github.com/prettier/prettier/issues/1080

I'm not precious about it anyway, it's probably more of an aesthetic decision than anything, and any kind of codified formatting is better than none.

Reply:
Yeah, I think single quote consistency is better and is already the style the codebase via standard.

Reply:
Added jsxSingleQuote and updated.  We'll probably need to ditch all the non-config changes and recreate the PR when it is time to merge anyway; the formatting changes in this PR are just to show what the formatted code would look like with these settings.